### DISCUSSION TITLE
Paid windows version without Windows Store?

### DISCUSSION THREAD

**Discussion by Wuff-Bellmann** [theophanemayaud/video-simili-duplicate-cleaner#118] [General]
Hi,

since I'm too stupid to build it on my own and since I don't have Windows store (and don't want to install it) is there a chance getting the paid Windows version any other way?

Thanks!

**Comment by theophanemayaud** [comment]
Hey, thanks for the interest ! Im sorry though, I don’t have the binary on hand and for the next few weeks, access to a windows computer to build it. If you’re still interested then I can send you the binary and a PayPal link 🤷‍♂️💕

**Reply by Wuff-Bellmann** [reply]
Thanks for your reply.

I tried to build it by myself today for several hours and wasn't successful. So, yes, I'm still interested, haha :)

Do you need my e-Mail?

**Reply by theophanemayaud** [reply]
I’ll set a reminder for when I have access to a windows machine in mid October again. I’ll contact you again then 👌

**Reply by theophanemayaud** [reply]
Hey, sorry for the delay, but do you still need the app :)

**Reply by Wuff-Bellmann** [reply]
Thanks, but I found a solution for me so actually I don't need it.
